examples/wCCD/src/wallet/WithWalletConnection.tsx

```diff
@@ -112,20 +131,15 @@ export class WithWalletConnection extends React.Component<Props, State> implemen
     };
 
     onChainChanged = (connection: WalletConnection, genesisHash: string) => {
-        console.debug('WithWalletConnection: onChainChanged called', { connection, genesisHash, state: this.state });
-        const { network } = this.props;
-        // Check if the user is connected to expected network by checking if the genesis hash matches the expected one.
-        // Emit a warning and disconnect if it's the wrong chain.
-        if (genesisHash !== network.genesisHash) {
-            window.alert(
-                `Unexpected genesis hash '${genesisHash}'. Expected '${network.genesisHash}' (network '${network.name}').`
-            );
-            connection.disconnect().catch(console.error);
+        console.debug("WithWalletConnection: calling 'onChainChanged'", { connection, genesisHash, state: this.state });
+        const { activeConnection } = this.state;
+        if (connection === activeConnection) {
+            this.setState({ activeConnectionGenesisHash: genesisHash });
         }
     };
 
     onDisconnect = (connection: WalletConnection) => {
-        console.debug('WithWalletConnection: onDisconnect called', { connection, state: this.state });
+        console.debug("WithWalletConnection: calling 'onDisconnect'", { connection, state: this.state });
```

Comment:
Can we remove the logs before this is deployed to the production testnet site? If this is intended to be published when merged, I think we should remove them now.

Reply:
I was intending to leave them in to be able to debug issues that inevitably happen. Why would you like to remove them? It's low volume logging, mostly on the debug level.

I'll merge it like this but not deploy for now.